crates/cargo-util-schemas/src/manifest/mod.rs

```diff
@@ -173,6 +173,7 @@ pub struct TomlPackage {
     pub publish: Option<InheritableVecStringOrBool>,
     pub workspace: Option<String>,
     pub im_a_teapot: Option<bool>,
+    pub autolib: Option<bool>,
```

Comment:
Everything looks, except this could potentially break other third-party build tools like [`cargo-manifest`](https://crates.io/crates/cargo-manifest) in the future. I'd like to give them a heads-up in <https://rust-lang.zulipchat.com/#narrow/stream/334885-t-cargo.2Fbuild-integration> and merge on Friday. Any other opinion?

Reply:
Posted: https://rust-lang.zulipchat.com/#narrow/stream/334885-t-cargo.2Fbuild-integration/topic/New.20.60package.2Eautolib.60.20for.20disabling.20auto-discovery.20for.20.5Blib.5D

Reply:
cc @kornelski and @LukeMathWalker since you're maintainers of relevant crates.

Reply:
cc @Turbo87 who is covering for me while I'm on pat leave

Reply:
yep yep, already aware of it :)